jina/peapods/pods/compoundpod.py

```diff
@@ -137,11 +74,14 @@ def num_peas(self) -> int:
 
         :return: total number of peas including head and tail
         """
-        return sum([replica.num_peas for replica in self.replica_list]) + len(self.peas)
+        return sum([replica.num_peas for replica in self.replicas]) + 2
```

Comment:
+2 means head and tail pea?

Reply:
yes

Reply:
`CompoundPod` is a special kind of Pod containing many Pod inside and surrounded by `head` and `tail`.

This one will ALWAYS have Head and Tail